ci-operator/config/securesign/secure-sign-operator/securesign-secure-sign-operator-main.yaml

```diff
@@ -35,6 +35,8 @@ operator:
   substitutions:
   - pullspec: registry.redhat.io/rhtas-tech-preview/sigstore-rhel9-operator.*
```

Comment:
maybe we should update this line too, because for GA the `preview` word will be removed

Reply:
do we have official coordinates for GA yet? is `registry.redhat.io/rhtas-tech/sigstore-rhel9-operator.*` correct?

Reply:
@lance Do you Know what the final coords are going to be for GA?

Reply:
@JasonPowr it should be `registry.redhat.io/rhtas/rhtas-rhel9-operator`